src/sentry/static/sentry/app/views/settings/account/accountSettingsLayout.jsx

```diff
@@ -1,9 +1,31 @@
 import React from 'react';
 
 import AccountSettingsNavigation from 'app/views/settings/account/accountSettingsNavigation';
+import {fetchOrganizationDetails} from 'app/actionCreators/organizations';
+import SentryTypes from 'app/sentryTypes';
 import SettingsLayout from 'app/views/settings/components/settingsLayout';
+import withLatestContext from 'app/utils/withLatestContext';
 
 class AccountSettingsLayout extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  componentDidUpdate(prevProps) {
+    let {organization} = this.props;
+    if (prevProps.organization === organization) return;
+
+    // if there is no org in context, SidebarDropdown uses an org from `withLatestContext`
+    // (which queries the org index endpoint instead of org details)
+    // and does not have `access` info
+    if (organization && !organization.access) {
```

Comment:
I think `access` is always an array, but we could be a bit more clear and check that `access` is explicitly not defined on `organization` instead of a general falsey value.